api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/health/ApiCatalogHealthIndicator.java

```diff
@@ -22,19 +24,40 @@
  */
 @Component
 @RequiredArgsConstructor
+@Slf4j
 public class ApiCatalogHealthIndicator extends AbstractHealthIndicator {
+    private static String AUTHENTICATION_SERVICE_ID = "apiml.authorizationService.zosmfServiceId";
```

Comment:
Is it safe to strictly link to specific authentication provider?
In future plans we should support SSO, wouldn't this have impact here?

Reply:
I see that this is just a link to configuration. As such this is going to be fine. 
When I was going through the code, I was confused why add these parameters when there are already the parameters:

apiml:
    security:
        auth:
            provider: dummy
            zosmfServiceId: zosmf  # Replace me with the correct z/OSMF service id